cves/kernel/CVE-2015-0275.yml

```diff
@@ -1,483 +1,585 @@
+
+
+   Example interesting commits:
+     * Mentioned as a problematic commit in the past
+       e.g. "This fixes regression in commit xys"
+     * A significant rewrite in the git history
+     * Other commits that fixed a similar issue as this vulnerability
+     * Anything else you find interesting.
+ commits:
+ - commit: e1ee60fd89670da61b0a4bda59f8ffb2b8abea63
+   note: |
+     This commit fixes an error when mounting ext4 with data = journal, and it was
+     related directly to fallocate zero flag functionality.
+ - commit: 713e8dde3e71e92db2d8cc8459d236ce1fb576ce
+   note: |
+     This commit fixes another zero range bug due to flag aliasing which could
+     cause a corruption in the data file. Its in the same area of the code as
+     this vulnerability.
 i18n:
-  question: |
-    Was the feature impacted by this vulnerability about internationalization
-    (i18n)?
-
-    An internationalization feature is one that enables people from all
-    over the world to use the system. This includes translations, locales,
-    typography, unicode, or various other features.
-
-    Answer should be true or false
-    Write a note about how you came to the conclusions you did, regardless of
-    what your answer was.
-  answer:
-  note:
+ question: |
+   Was the feature impacted by this vulnerability about internationalization
+   (i18n)?
+
+
+   An internationalization feature is one that enables people from all
+   over the world to use the system. This includes translations, locales,
+   typography, unicode, or various other features.
+
+
+   Answer should be true or false
+   Write a note about how you came to the conclusions you did, regardless of
+   what your answer was.
+ answer: false
+ note:  |
+   This error was caused by a bug with allocating file space, i18n did not
+   play a factor in causing the issue.
 sandbox:
-  question: |
-    Did this vulnerability violate a sandboxing feature that the system
-    provides?
-
-    A sandboxing feature is one that allows files, users, or other features
-    limited access. Vulnerabilities that violate sandboxes are usually based on
-    access control, checking privileges incorrectly, path traversal, and the
-    like.
-
-    Answer should be true or false
-    Write a note about how you came to the conclusions you did, regardless of
-    what your answer was.
-  answer:
-  note:
+ question: |
+   Did this vulnerability violate a sandboxing feature that the system
+   provides?
+
+
+   A sandboxing feature is one that allows files, users, or other features
+   limited access. Vulnerabilities that violate sandboxes are usually based on
+   access control, checking privileges incorrectly, path traversal, and the
+   like.
+
+
+   Answer should be true or false
+   Write a note about how you came to the conclusions you did, regardless of
+   what your answer was.
+ answer: false
+ note: |
+   the vulnerability was related to allocating file space, and did not violate
```

Comment:
Based on your research, is it possible to exploit this vulnerability to expose privileged data? For example, something else is in the stack, and the zero-ranged memory is smaller than what is required, which causes overflow and dumps a block of stack data?